### PR TITLE
Move Localization to Node

### PR DESCRIPTION
**Pull request by NathanFlurry**

### CODE REVIEW

File: Sources/Vapor/Localization/Localization.swift [hunk 1]
```diff
@@ -1,26 +1,52 @@
...
-        let localization = try FileManager.loadDirectory(configDirectory)
-        self.init(jsonDirectory: localization)
+    public convenience init(workingDirectory: String, path: String? = nil) throws {
+        let localizationDirectory = workingDirectory.finished(with: "/") +  (path ?? "Localization/")
+        try self.init(path: localizationDirectory)
+    }
+    
+    public convenience init(path: String) throws {
+        // List the files in the directory or have no localizations if could not find files
+        guard let contents = try? FileManager.contentsOfDirectory(path) else {
```
Comment (tannernelson) [inline Sources/Vapor/Localization/Localization.swift]: Remove `?` from try so the throw keeps going.

**Comment by tannernelson** [comment]
How come test coverage went down? 0_o

Is there any way we can add some more coverage? Otherwise, this looks good to me and we can merge whenever!

File: Sources/Vapor/Localization/Localization.swift (hunk 1 above)
Comment (NathanFlurry) [inline Sources/Vapor/Localization/Localization.swift]: I thought so too originally, but having the Localization directory should be optional in all projects. If the user does not provide localizations, the program should still run just fine. Many web servers won't have localizations. Maybe we should move this logic to the Droplet?

Comment (NathanFlurry) [inline Sources/Vapor/Localization/Localization.swift]: Oh it looks like there's already something in the Droplet for this. I guess one of the tests was messed up, I'll go fix that.

**Comment by tannernelson** [comment]
Thanks!

**Pull request merged**